src/integration-test/java/se/sundsvall/supportmanagement/apptest/OpenApiSpecificationIT.java

```diff
@@ -0,0 +1,88 @@
+package se.sundsvall.supportmanagement.apptest;
+
+import static net.javacrumbs.jsonunit.assertj.JsonAssertions.assertThatJson;
+
+import java.util.List;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.dataformat.yaml.YAMLMapper;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.core.io.Resource;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.web.util.UriComponentsBuilder;
+
+import se.sundsvall.dept44.util.ResourceUtils;
+import se.sundsvall.supportmanagement.Application;
+
+import net.javacrumbs.jsonunit.core.Option;
+
+@ActiveProfiles("it")
```

Comment:
Is IT the correct profile for this?
(IT == integration test ??) 

This code tests the OpenAPI-definition of this service?